### PR TITLE
ChangeLog for 202006.00 release

### PR DESCRIPTION
**Pull request by aggarw13**
Add ChangeLog for `202006.00` release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+
+- New Board: <b>ESP32-WROOM_32-SE</b> is now qualified by FreeRTOS. It was earlier supported in Preview mode before.
+- Support OTA data over HTTP together with BLE on ESP32 when SPIRAM is enabled.
+- Added ESP-IDF component for WiFi provisioning in SoftAP mode. This allows WiFi provisioning of devices through a web-server running on the device and a provisioning mobile application to enter WiFi credentials onto the device for provisioning. This requires use of lwIP as the networking stack.
+
+## Updates
+
+### FreeRTOS+POSIX Utils V1.2.0
+
+- Update UTILS_TimespecAdd POSIX utility function to support both signed and unsigned definitions of struct timespec.tv_sec. (Some implementations use unsigned definition of time_t to address the *2038* problem on 32-bit systems. ) This change is backwards compatible.
```
Comment (aggarg) [inline CHANGELOG.md:21]: Some implementations use unsigned definition of time_t --> Some implementations use unsigned definition of struct timespec.tv_sec

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+
+### PKCS#11
+
+- Added doxygen to various PKCS #11 files.
+- Added improved logging for mbed TLS  return codes in iot_pkcs11_mbedtls.c. 
+
+### Bluetooth Low Energy Management Library V2.1.0
+
+- Added new API IotBle_SetDeviceName() to set the BLE device name at runtime.
+- Made fixes to IotBle_On() and IotBle_Off() APIs
```
Comment (aggarg) [inline CHANGELOG.md:36]: End with period.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+- Upgraded the version of Mbed TLS to v2.16.6.
+- Submoduled the Mbed TLS library.
+
+### Over the Air Update V1.2.0
+
+- Fixed an issue where self-test timer is started after receiving the AWS IoT Job document. The fix makes the timer start before any job document parsing is done.
+
+### Espressif
+
+- Replaced ESP-IDF code to be a submodule pointer to the official Espressif-ID repository.
```
Comment (aggarg) [inline CHANGELOG.md:54]: official *Espressif-ID* or IDF?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+- Added ability to cache multiple IP addresses per DNS entry.
+
+### Mbed TLS v2.16.6
+
+- Upgraded the version of Mbed TLS to v2.16.6.
+- Submoduled the Mbed TLS library.
+
+### Over the Air Update V1.2.0
+
+- Fixed an issue where self-test timer is started after receiving the AWS IoT Job document. The fix makes the timer start before any job document parsing is done.
```
Comment (aggarg) [inline CHANGELOG.md:50]: The two sentences do not connect. Do you mean to say:

Fixed an issue where self-test timer was started after parsing the AWS IoT Job document. The fix makes the timer start before any job document parsing is done.

File: CHANGELOG.md (hunk 3 above)
Comment (aggarw13) [reply CHANGELOG.md:54]: Will `esp-idf` instead: https://github.com/espressif/esp-idf

File: CHANGELOG.md (hunk 1 above)
Comment (aggarw13) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+
+### Over the Air Update V1.2.0
+
+- Added feature to suspend and resume OTA Agent using new APIs provided. This allows OTA Agent to resume download after suspending it for any tasks like re-establishing MQTT connection. 
+- OTA Demo is updated to check for MQTT disconnect, suspend OTA operations and try reconnecting with exponential delay and jitter. After successful connection the demo resumes OTA Agent operations.
+- Added new OTA config for allowing downgrade and version update. This is provided for testing purpose and configuration is disabled by default.
+
+### Espressif
+
+- New Board: <b>ESP32-WROOM_32-SE</b> is now qualified by FreeRTOS. It was earlier supported in Preview mode before.
```
[the remark this answers is not in the document]
Comment (aggarw13) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 4 above)
Comment (aggarw13) [reply CHANGELOG.md:50]: How about using the following text:
"_Fixed an issue for the scenario when an OTA job is force cancelled when its related download is in progress, caused due to the self-start timer starting after the OTA job document is received. The fix makes the self-start timer start when the OTA agent on the device starts, before receiving the OTA job._"

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,59 @@
...
+
+- New Board: <b>ESP32-WROOM_32-SE</b> is now qualified by FreeRTOS. It was earlier supported in Preview mode.
+- Support OTA data over HTTP together with BLE on ESP32 when SPIRAM is enabled.
+- Added ESP-IDF component for WiFi provisioning in SoftAP mode. This allows WiFi provisioning of devices through a web-server running on the device and a provisioning mobile application to enter WiFi credentials onto the device for provisioning. This requires use of lwIP as the networking stack.
+
+## Updates
+
+### FreeRTOS+POSIX Utils V1.2.0
+
+- Update UTILS_TimespecAdd POSIX utility function to support both signed and unsigned definitions of `struct timespec.tv_sec`. (Some implementations use unsigned definition of `struct timespec.tv_sec` to address the *2038* problem on 32-bit systems. ) This change is backwards compatible.
```
Comment (yuhui-zheng) [inline CHANGELOG.md:21]: Do we want to announce none API changes in CHANGELOG.md? ```UTILS_TimespecAdd``` this is not POSIX API, but rather utilities used in POSIX API implementation.

Comment (aggarw13) [reply CHANGELOG.md:21]: The text mentions the function as a utility. As these utilities are exposed in the `libraries/abstractions/posix/` folder and if a customer is using any of these utilities in their application, I think it is important to mention the update

**Pull request merged**